libminifi/src/sitetosite/SiteToSiteClient.cpp

```diff
@@ -499,6 +505,9 @@ int16_t SiteToSiteClient::send(std::string transactionID, DataPacket *packet, co
       return -1;
     }
     packet->_size += len;
+  } else if (flowFile && !flowfile_has_content) {
+    uint64_t len = 0;
```

Comment:
This shadows the outer len.

Reply:
Ouch! Fair point, especially considering the fact that I need the same default value here :)